src/EnergyPlus/Plant/LoopSide.cc

```diff
@@ -926,16 +938,16 @@ namespace DataPlant {
                             // special primary side flow request for two way common pipe
                             int const CompIndex = component.CompNum;
                             {
-                                auto const SELECT_CASE_var(component.TypeOf_Num);
```

Comment:
`Switch/case`. The fact that SELECT_CASE_var is here tells you that this was a `switch/case` in the original Fortran code.